cli.js

```diff
@@ -123,7 +123,13 @@ const promptEngines = () => {
 
 	const args = process.argv.slice(2);
 	for (const arg of args) {
-		if (arg.startsWith('--os=')) {
+		if (arg.startsWith('--help') || arg.startsWith('-h')) {
```

Comment:
Why `startsWith`? We can be strict in this case:

```js
if (arg === '--help' || arg === '-h') {
  // ...
}
```

Reply:
My bad :) I was too concerned by syntax & co (so i'v copy/paste your code ^^).